src/less/tooltip/base/tooltip.less

```diff
@@ -141,16 +143,23 @@ span.tourtip {
     display: inline-block;
 }
 
+.tooltip__overlay,
+.infotip__overlay {
+    left: 0 - @icon-width/2;
+    margin-top: 11px; // half the height of the pointer
```

Comment:
Should we use the `@icon-width` variable here?

Reply:
The distance from the top is really more about the height of the pointer.  I was tempted to do something like this (rather than use margin):
```
top: @icon-height + 11px;
```
But its already naturally positioned after the icon.
What do you think?

Reply:
I think it's fine, but I wanted to keep consistency between the left declaration which used `@icon-width` and the top margin which did not use `@icon-height`.